lib/tasks/thredded_tasks.rake

```diff
@@ -5,4 +5,14 @@ namespace :thredded do
   end
 
   task :nuke, [:slug] => :destroy
+
+  namespace :install do
+    desc 'Copy emoji to the Rails `public/emoji` directory'
+    task :emoji do
+      require 'emoji'
```

Comment:
The gemoji gem [already provides a rake task](https://github.com/github/gemoji#gemoji) for this, so we can just do:

```ruby
load 'tasks/emoji.rake'
Rake::Task['emoji'].invoke
```